charts/sysdig/templates/_helpers.tpl

```diff
@@ -295,3 +295,16 @@ true
 true
 {{- end -}}
 {{- end -}}
+
+{{/* Returns string 'true' if the cluster's kubeVersion is less than the parameter provided, or nothing otherwise
+     Use like: {{ include "sysdig.kubeVersionLessThan" (dict "root" . "major" "<kube_major_to_compare>" "minor" "<kube_minor_to_compare>") }}
```

Comment:
```suggestion
     Use like: {{ include "sysdig.kubeVersionLessThan" (dict "root" . "major"<kube_major_to_compare> "minor" <kube_minor_to_compare>) }}
```
major and minor should be number, adding quotes might sound like we need to indicate them as string but then we are not casting them as integer into template..

Reply:
All set. 🙂